simulator/src/data/project.js

```
@@ -58,7 +58,12 @@ export function openOffline() {
             text: 'Open Project',
             click() {
                 if (!$('input[name=projectId]:checked').val()) return;
-                load(JSON.parse(localStorage.getItem($('input[name=projectId]:checked').val())));
+                const data = JSON.parse(localStorage.getItem($('input[name=projectId]:checked').val()));
+                const simulatorVersion = data.simulatorVersion             
```

ESLint

[error] Use object destructuring.

(prefer-destructuring)